content/_data/changelogs/lts.yml

```
@@ -6672,6 +6672,263 @@
+    authors:
+      - benebsiny
+    pr_title: "[JENKINS-67496] Drag & drop is messed up, drag placeholders always\
+      \ have tiny size"
+    message: |-
+      Keep the same height when dragging and dropping a component (regression in 2.277).
+
+  - type: bug
+    category: bug
+    pull: 6225
+    issue: 67627
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-67627] Have expandable text boxes expand into multiple\
+      \ lines"
+    message: |-
+      Correctly render expandable text boxes into multiple lines (regression in 2.197 and 2.176.4).
+
+  - type: bug
+    category: bug
+    pull: 6206
+    issue: 67662
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-67662] Do not show wrong feature name in tooltips"
+    message: |-
+      Show correct feature name in tooltips of help links (regression in 2.179).
+
+  - type: bug
+    category: regression
+    pull: 6193
+    issue: 67635
+    authors:
+      - NivKeidan
+    pr_title: JENKINS-67635 consider agent label expressions when applying trimLabels
+    message: |-
+      Launch only one agent to satisfy cloud agent requests that use label expressions.
+
+  - type: bug
+    category: bug
+    pull: 6234
+    issue: 67689
+    authors:
+      - timja
+    pr_title: JENKINS-67689 Don't hide cancel button with long build names
+    message: |-
+      Truncate long build names again (regression in 2.332).
+
+  - type: bug
+    category: bug
+    pull: 6264
+    issue: 67753
+    authors:
+      - NotMyFault
+    pr_title: '[JENKINS-67753] Overwrite "grey" balls with modern build status'
+    message: |-
+      Overwrite grey balls icon with the modern "not built" status.
+
+  - type: bug
+    category: regression
+    pull: 6261
+    authors:
+      - dependabot[bot]
+    pr_title: "[JENKINS-67000] Bump remoting from 4.11.2 to 4.12"
+    references:
+      - pull: 5983
+      - issue: 67000
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.11.2
+        title: Remoting 4.11.2 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.12
+        title: Remoting 4.12 changelog
+    message: |-
+      Update remoting from 4.11 to 4.12 to allow Java web start agents to connect (regression in 2.318).
+
+  lts_changes: # compared to lts_predecessor 2.319.3 (selected by personal review)
+
+  # Agent to controller subsystem is now always enabled
+
+  - type: major rfe
+    category: major rfe
+    pull: 5885
+    issue: 67173
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-67173] Remove Callable allowlist and FilePath agent-to-controller\
+      \ support"
+    message: |-
+      The agent-to-controller security subsystem is now always enabled.
+      The admin-customizable allowlists for callables and file paths have been removed.
+      The ability to access some files on the controller from agents has been removed.
+      Some plugins are incompatible with this change and may need to be updated.
```

I feel like the last sentence should also be updated but am at sort of a loss.  @StackScribe  maybe?
```suggestion
      Always enable the agent-to-controller security subsystem.
      Remove the admin-customizable allowlists for callables and file paths.
      Remove the ability to access some files on the controller from agents.
      Use the Plugin Manager to upgrade incompatible plugins.
```